src/components/Button.tsx

```diff
@@ -115,54 +127,58 @@ export function Button(props: ButtonProps) {
   });
 }
 
-function getButtonStyles(variant: ButtonVariant, size: ButtonSize) {
+function getButtonStyles(variant: ButtonVariant, size: ButtonSize, contrast: boolean) {
+  const styles = variantStyles(contrast)[variant];
   if (variant === "text") {
     // The text variant does not support the 'size'. The 'size' prop only effects the button's height and padding which is not relevant for this variant.
-    return variantStyles[variant];
+    return styles;
   }
   return {
-    ...variantStyles[variant],
-    baseStyles: { ...variantStyles[variant].baseStyles, ...sizeStyles[size] },
+    ...styles,
+    baseStyles: { ...styles.baseStyles, ...sizeStyles[size] },
   };
 }
 
-const variantStyles: Record<ButtonVariant, { baseStyles: {}; hoverStyles: {}; disabledStyles: {}; pressedStyles: {} }> =
-  {
-    primary: {
-      baseStyles: Css.bgLightBlue700.white.$,
-      hoverStyles: Css.bgLightBlue900.$,
-      pressedStyles: Css.bgLightBlue500.$,
-      disabledStyles: Css.bgLightBlue200.$,
-    },
+const variantStyles: (
+  contrast?: boolean,
```

Comment:
Do we want `contrast` to be an optional params that callers can forget to pass?

Reply:
yea, probably not necessary to be optional here. Thanks for pointing out.